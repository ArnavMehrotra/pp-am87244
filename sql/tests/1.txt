A B C
1 2 -10
2.3 5.1 300
210.15 211.13 -60.13
