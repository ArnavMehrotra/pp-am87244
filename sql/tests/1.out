C
-10
300
-60.13
